### ISSUE TITLE
A11y_Model Builder_Advance data option_Column Settings_Status Messages : Narrator/NVDA is not announcing the "No Column Found" status message when no column is found for the search.

### ISSUE THREAD

**Issue by manaswita-chougule** [dotnet/machinelearning-modelbuilder#2681]
### GitHub Tags
#WCAG4.1.3; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #Status Messages; #Narrator; #NVDA; #Win32;

### Environment
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22621.1702
Narrator
NVDA : Version 2023.1

### Repro Steps

1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Complete Scenario and environment setup steps.
7. Add Data screen will appear. Tab till Advance data option.
8. Advance data option screen will appear, tab till Column Setting and press enter.
9. Turn on Narrator/NVDA.
10. Tab till Search field and enter any random search.
11. Observe that Narrator/NVDA is not announcing the "No Column Found" status message when no column is found for the search.

### Actual Result
Narrator/NVDA is not announcing the "No Column Found" status message when no column is found for the search.

### Expected Result
Narrator/NVDA should announce the "No Column Found" status message when no column is found for the search.

### User Impact
Screen reader users will not able to get the status about the specified search if the status message is not being announced by screen reader.

### Attachment
Narrator
![Narrator_Status message](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/5567f4cb-8a2f-4631-a6f8-9bbe2c777daa)

NVDA
![NVDA_status messages](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/24366388-8696-4219-85d6-b26253067707)

**Comment by zewditu** [comment]
@manaswita-chougule  , ready to validate by Model Builder version 16.17.0.2332602.

**Comment by manaswita-chougule** [comment]
#Regressed:06-29-23; #Closed;
Verified the issue in below environment:
Visual Studio : Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.6.3
Model Builder : Version 17.17.0.2332602

The issue is fixed, hence closing the bug. Please find attachment below for reference. Thanks!
Narrator:
![Narrator is announcing the No Column Found status message when no column is found for the search](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/55239fb8-b2cc-4c01-995d-8463f88dc13a)

NVDA:
![NVDA is announcing the No Column Found status message when no column is found for the search](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/986ae6f3-f237-430a-9e22-c3438cea9ce5)
